src/components/Home/TreasuryOverview/index.tsx

```diff
@@ -297,7 +297,7 @@ const TreasuryOverview: FC<ITreasuryOverviewProps> = (props) => {
 				});
 				const responseJSON = await response.json();
 				if (responseJSON['message'] == 'Success') {
-					const weekAgoPrice = responseJSON['data']['ema7_average'];
+					const weekAgoPrice = responseJSON['data']['list'][0]?.['price'] ? responseJSON['data']['list'][0]['price'] : responseJSON['data']['ema7_average'];
```

Comment:
make it responseJSON?.['data']?.['list']?.[0]?.['price'] ? responseJSON?.['data']?.['list']?.[0]?.['price'] : responseJSON?.['data']